packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/ExpandCollapseFolderTree.jsx

```diff
@@ -28,11 +28,31 @@ const ExpandCollapseFolderTree = () => {
     setExpandedKeys(newExpandedKeys);
   };
 
+  const handleExpandAll = () => {
+    const newExpandedKeys = {
```

Comment:
Suggestion:
It might just be me but this naming pattern threw me off for a minute. `expandedKeys` and `newExpandedKeys` sound to me like they contain _'a list of keys of expanded items'_ and _'a list of keys of items to be expanded'_, rather than _'an object containing the expansion state for all the items'_ and _'a copy of that object on which to update the state'._ 

Since this example is part of documentation and these functions will likely be copied, verbatim, into production code, I think they could benefit from being clearer. Something like `itemsExpanded` and `itemsExpandedCopy` or `newItemsExpanded`...or something better you come up with.

`selectedKey` makes sense because it's literally the key of the selected item.

Reply:
Updated to `expandedItems`, does that sound okay? [d1bad61](https://github.com/cerner/terra-framework/pull/1943/commits/d1bad61e7af821395a82e2f4a6a8e8481994e127)

Reply:
Still a lil off since it also contains collapsed items, but I can't think of a better name. Not a blocker or anything.